x-pack/plugins/security_solution/public/management/pages/event_filter/view/components/modal/translations.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const MODAL_TITLE = i18n.translate('xpack.securitySolution.eventFilter.modal.title', {
+  defaultMessage: 'Add Endpoint Event Filter',
```

Comment:
We should check what the proper capitilization should be. in some cases, I see any word after the first one be all lowercase (Endpoint should still be upper I think). so should this be `Add Endpoint event filter`? (same comemnt for others below

Reply:
I used the ones on this [screenshot](https://user-images.githubusercontent.com/56395104/112144502-ae9d1580-8baf-11eb-8a61-9ef808da636b.png) but agree that should ensure that.
cc: @caitlinbetz @kevinlog